contracts/azorius/Azorius.sol

```diff
@@ -168,13 +158,16 @@ contract Azorius is Module, IAzorius {
         totalProposalCount++;
     }
 
-    /// @notice Executes the specified transaction within a proposal
-    /// @notice Transactions must be called in order
-    /// @param _proposalId the identifier of the proposal
-    /// @param _target the contract to be called by the avatar
-    /// @param _value ether value to pass with the call
-    /// @param _data the data to be executed from the call
-    /// @param _operation Call or Delegatecall
+    /**
+     * Executes the specified transaction in a Proposal, by index.
+     * Transactions in a proposal must be called in order.
+     *
+     * @param _proposalId identifier of the proposal
+     * @param _target contract to be called by the avatar
+     * @param _value ETH value to pass with the call
+     * @param _data data to be executed from the call
+     * @param _operation Call or Delegatecall
+     */
```

Comment:
would a more accurate name be `executeProposalTxByIndex`?

Reply:
Yeah, agreed I think that's a better description

Reply:
@herbig Do you want to address this function name change in this PR, or would you like me to change it my current branch?

Reply:
I can update this, no prob

Reply:
Oh yeah, also making this private will cause some tests to fail. I can push commit to fix those on this branch unless you want to tackle it

Reply:
Ah true, yeah might be quicker for you to fix those: https://github.com/decent-dao/fractal-contracts/pull/35/commits/662162003dd69fa264b3d74204d79de8ceb52c01

Reply:
This is done